### PR TITLE
[IMP] fieldservice: Give Access to portal user

### PR DESCRIPTION
**Pull request by murtuzasaleh**

### CODE REVIEW

**Review by marcelsavegnago — APPROVED** [review]

File: fieldservice/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,32 +1,53 @@
...
 access_fsm_order_type_user,fsm.order.type.user,model_fsm_order_type,fieldservice.group_fsm_user,1,1,0,0
 access_fsm_order_type_manager,fsm.order.type.manager,model_fsm_order_type,fieldservice.group_fsm_manager,1,1,1,1
+access_fsm_order_type_portal,access.fsm.order.type.portal,model_fsm_order_type,base.group_portal,1,0,0,0
 access_fsm_calendar_filter,fsm.calendar.filter.user,model_fsm_person_calendar_filter,fieldservice.group_fsm_user,1,1,1,1
+access_resource_resource_portal,access.resource.resource.portal,resource.model_resource_resource,base.group_portal,1,0,0,0
+access_ir_sequence_portal,access.ir.sequence.portal,base.model_ir_sequence,base.group_portal,1,0,0,0
+access_res_partner_bank_portal,access.res.partner.bank.portal,base.model_res_partner_bank,base.group_portal,1,0,0,0
+access_res_partner_portal,access.res.partner.portal,base.model_res_partner,base.group_portal,1,0,0,0
+access_ir_attachment_portal,access.ir.attachment.portal,base.model_ir_attachment,base.group_portal,1,0,0,0
+access_mail_message_portal,access.mail.message.portal,mail.model_mail_message,base.group_portal,1,0,0,0
```
Comment (brian10048) [inline fieldservice/security/ir.model.access.csv]: Is there a reason we need to add these permissions for models not in fieldservice?

Comment (max3903) [reply fieldservice/security/ir.model.access.csv]: @brian10048 We are using the portal group to setup access from mobile app. I believe @murtuzasaleh got a permission error without this change.

Comment (murtuzasaleh) [reply fieldservice/security/ir.model.access.csv]: @brian10048 I got a permission error in the mobile app for the portal group.

Comment (murtuzasaleh) [reply fieldservice/security/ir.model.access.csv]: @brian10048 removed unused access rights.

**Review by max3903 — APPROVED** [review]

**Pull request merged**